scripts/pca_minimal.py

```diff
@@ -0,0 +1,119 @@
+#!python3
```

Comment:
Do we want to include a module-level docstring and a description of the main parameters here? I guess this is the responsibility of the person who wrote the original `pca` script and/or me!

Reply:
resolved in 1c44bdf59b6635e9020684bf987fe5391dd25960